actioncable/actioncable.gemspec

```diff
@@ -18,6 +18,11 @@ Gem::Specification.new do |s|
   s.files        = Dir["CHANGELOG.md", "MIT-LICENSE", "README.md", "lib/**/*"]
   s.require_path = "lib"
 
+  s.metadata = {
+    "source_code_uri" => "https://github.com/rails/rails/tree/master/actioncable",
+    "changelog_uri"   => "https://github.com/rails/rails/blob/master/actioncable/CHANGELOG.md"
```

Comment:
I'm a little bit worried on pointing the changelog to the master branch. The correct changelongs are in the release branches so it make no sense to point to master. The same happens with the source code. Maybe it should point to the tag?